crypto/x509/x509_test.cc

```diff
@@ -2301,6 +2302,32 @@ TEST(X509Test, Ed25519Sign) {
   ASSERT_TRUE(SignatureRoundTrips(md_ctx.get(), pub.get()));
 }
 
+TEST(X509Test, DilithiumSignVerifyCert) {
+  //generate the dilithium key
+  EVP_PKEY_CTX *dilithium_pkey_ctx = EVP_PKEY_CTX_new_id(EVP_PKEY_DILITHIUM3, nullptr);
```

Comment:
nit: can we get rid of the `dilithium_` prefix in the variables to make the code more concise (we already now it's Dilithium because of the test name...)?

Reply:
[3f25157](https://github.com/awslabs/aws-lc/pull/759/commits/3f25157de0e30dbefa578cddf4a193129935b9be)